Tests/Unit/EmogrifierTest.php

```diff
@@ -1766,6 +1766,19 @@ public function removeExcludedSelectorGetsMatchingElementsToBeEmogrifiedAgain()
         self::assertContains('<p class="x" style="margin: 0;"></p>', $result);
     }
 
+    /**
+     * @test
+     *
+     * @expectedException \InvalidArgumentException
```

Comment:
Please use 'expectException' instead. It's the recommended way now: https://phpunit.de/manual/current/en/writing-tests-for-phpunit.html#writing-tests-for-phpunit.exceptions

(I've created issue #438 for converting the other tests.)